README.md

```diff
@@ -364,6 +364,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Geo](https://github.com/davidmoten/geo/) - GeoHash utilities in Java.
 * [Geotoolkit.org](http://www.geotoolkit.org/) - Library for developing geospatial applications. Built on top of the Apache SIS project.
 * [GeoTools](http://geotools.org/) - Library that provides tools for geospatial data.
+* [GraphHopper](https://github.com/graphhopper/graphhopper) - Apache licensed road routing engine using OpenStreetMap data.
```

Comment:
Please be concise as possible and try to emulate the general style of the descriptions, i.e. License is not necessary, so all in all `Road routing engine.` is sufficient. You could mention that it uses OpenStreetMap data as default but I don't think this additional information is advantageous for the reader, do you?